share/robot.py

```diff
@@ -166,3 +206,21 @@ def __call__(self, apps, schema_editor):
             task=self.config.task,
             args=json.dumps([self.config.label, 1]),  # Note 1 should always be the system user
         ).update(enabled=False)
+
+
+class RobotFaviconMigration(AbstractRobotMigration):
+
+    def __call__(self, apps, schema_editor):
+        user = self.config.user
+        try:
+            with open(os.path.join(self.config.path, 'favicon.ico'), 'rb') as f:
+                user.favicon.save(user.username, File(f))
+        except OSError:
+            pass
```

Comment:
This exception shouldn't be caught.  Anything that has this migration should have a favicon as well.